server/game/scene.js

```diff
@@ -219,13 +226,21 @@ class GameScene extends Scene {
     levelMap.getObjectLayer('boxes')['objects'].forEach(box => {
       this.boxesGroup.add(new SpriteItems[box.name](this, this.getID(), box.x, box.y))
     })
+    // Add moving platforms
+    levelMap.getObjectLayer('moving_platform')['objects'].forEach(movingPlatform => {
+      const platform = new Platform(this, this.getID(), movingPlatform.x, movingPlatform.y)
+      this.movingPlatforms.add(platform)
+      platform.moveVertically()
+    })
 
     this.physics.add.collider(this.ingredientsGroup, worldLayer)
     this.physics.add.collider(this.playersGroup, worldLayer)
     this.physics.add.collider(this.ingredientsGroup, this.boxesGroup)
+    this.physics.add.collider(this.playersGroup, this.movingPlatforms, this.onMovingPlatformColliding);
```

Comment:
The moving platforms are really cool. Two notes:

- When I stand on the moving platform the player sprite keeps bouncing up down very slightly
- I noticed that if I throw a tomato on the platform it falls through.

Reply:
Yeah, the moving platform is still acting weird. I need to figure out that physics a bit better. As for the tomato falling through, I believe it's because I have not added a collision between those two.

Reply:
Okay, I fixed this issue and now you can throw veggies in the moving platforms. Also, the player bounces off when they jump on it because the player group has a bounceY configuration, but I believe there was also some unnecessary bouncing in the approach I took before, so hopefully that is resolved now.